### DISCUSSION TITLE
RADV Ray Tracing: Now ON by default | clusterduck - pixelcluster's GPU blog

### DISCUSSION THREAD

**Discussion by giscus** [pixelcluster/pixelcluster.github.io#5] [Comments]
# RADV Ray Tracing: Now ON by default | clusterduck - pixelcluster's GPU blog

https://pixelcluster.github.io/RADV-Raytracing-ON/

**Comment by peterdk** [comment]
Interesting read. I am wondering, do NVIDIA cards do more in hardware when raytracing? Or is their performance to be expected equal?

**Reply by pixelcluster** [reply]
I don't really know how much NVIDIA does in hardware actually. They're pretty secretive in these regards. My personal guess would be that hardware is at least a part of why they're ahead, but a lot of sophisticated stuff on the software side plays a considerable role as well.

**Reply by DiabeticCrab** [reply]
AFAIK the biggest differences stem from fixed-function ray traversal acceleration and the (optional, has to be implemented by the game devs) ability to do instruction reordering, which can compensate for the divergent nature of RT and the ALU utilization issues that stem from this.

Since AMD does traversal in compute, utilization is everything. One example for this would be Elden Ring, which on top of being run through VKD3D in my case, also only uses DXR1.0 (DXR1.1 can be layered more efficiently). My 6800XT pushes around 35FPS in 1440p, with drops below that depending on the scene, while running at "full" utilization, max clockspeed and drawing a max of 140W -> most of the chip is running idle. Astonishingly, it only makes a very small difference to performance when changing RT from "max" to "low", due to this game being barely bottlenecked by intersection throughput.

**Reply by jams3223** [reply]
I agree NVIDIA are very anti open-source unless money is involve, AMD has much more love for open source project, but most of the incredible work put in Linux has been from the Valve engineer, talents i never knew i could find who really have a passion for what they're doing, never seen a work environment where no one is pressured and everyone can make their time to work stress free, this is what makes them different from other companies. Friedrich i am proud of you man, without you in the boys people wouldn't know that the impossible is possible, if you need help to beta test i am here.
